docs/csharp/programming-guide/arrays/codesnippet/CSharp/passing-arrays-as-arguments_4.cs

```diff
@@ -1,68 +1,57 @@
-class ArrayClass
+using System;
+
+class ArrayExample
 {
     static void PrintArray(string[] arr)
     {
         for (int i = 0; i < arr.Length; i++)
         {
-            System.Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
+             Console.Write(arr[i] + "{0}", i < arr.Length - 1 ? " " : "");
         }
-        System.Console.WriteLine();
+        Console.WriteLine();
     }
 
     static void ChangeArray(string[] arr)
     {
-        // The following attempt to reverse the array does not persist when
-        // the method returns, because arr is a value parameter.
-        arr = (arr.Reverse()).ToArray();
-        // The following statement displays Sat as the first element in the array.
-        System.Console.WriteLine("arr[0] is {0} in ChangeArray.", arr[0]);
+        // // Reversing the array persists when the method returns.
+        Array.Reverse(arr);
     }
 
     static void ChangeArrayElements(string[] arr)
     {
-        // The following assignments change the value of individual array 
-        // elements. 
-        arr[0] = "Sat";
-        arr[1] = "Fri";
-        arr[2] = "Thu";
-        // The following statement again displays Sat as the first element
-        // in the array arr, inside the called method.
-        System.Console.WriteLine("arr[0] is {0} in ChangeArrayElements.", arr[0]);
+        // The following assignments change the value of individual array elements. 
+        arr[0] = "Mon";
+        arr[1] = "Wed";
+        arr[2] = "Fri";
     }
 
     static void Main()
     {
         // Declare and initialize an array.
         string[] weekDays = { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" };
-
-        // Pass the array as an argument to PrintArray.
+        // Display the array elements.
         PrintArray(weekDays);
```

Comment:
nit: `PrintArray` -> `DisplayArray` and the comment above the call can be removed.